Modules/rat_rescue.py

```diff
@@ -95,6 +96,15 @@ def __init__(self, case_id: UUID, client: str, system: str, irc_nickname: str,
         self._lang_id = lang_id
         self._status = status
 
+        # post-init cleanup
+        self.changedAttr.clear()
```

Comment:
lower case and underscores m8.
Yeet.
Is that how the kids talk nowadays?

Reply:
camelCase > underscore_delimited :^)

Reply:
I'm a bit on the fence.
I find lowercase with underscores more easily readable, but camelCase often shorter and quicker to type.

Reply:
But underscores are Python convention

Reply:
Yeah I'm mainly just joking around. :P

Reply:
I figured ;D

Reply:
honestly, i far prefer camelCase, but we are a bit too far down the snake_case rabit hole ;(